sdk/provision/chained_provisioner.go

```diff
@@ -0,0 +1,35 @@
+package provision
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+)
+
+// ChainedProvisioner chains multiple provisioners together for use at the same time.
+type ChainedProvisioner struct {
+	sdk.Provisioner
```

Comment:
Is there a need for the new chain provisioner to have a provisioner field? It implements the interface simply by implementing the required functions, so I feel this is not needed here.